docs/content/misc/contribute.ngdoc

```diff
@@ -61,7 +61,8 @@ To build AngularJS, you clone the source code repository and use Grunt to genera
 minified AngularJS files:
 
 ```shell
-# Clone your Github repository:
+# Clone your Github repository(Note: Do not go with `sudo` user when cloning into this repository 
```

Comment:
So should I include that and replace with the original Note (Do not go with `sudo` user when cloning into this repository ) ?